main.js

```diff
@@ -32,6 +32,14 @@ function createProduct(products) {
       $heartIcon.className += ' fav';
     }
   });
+
+  $cartPlus.addEventListener('click', function() {
+    if ($cartPlus.className !== 'fa fa-cart-plus in-cart') {
+      data.cart.push(products);
+      $cartPlus.className += ' in-cart';
+    }
+  });
+
```

Comment:
Now that you learned about event delegation, attempt to add one event listener on the product container, and handle your logic from there.